### PR TITLE
Arabic Translation Upload

### PR DESCRIPTION
**Pull request by alhitary**

### CODE REVIEW

**Comment by alhitary** [comment]
I'm sorry but I really made tries to add my Arabic Translation PR to the link but I could't or i don't know if i do it as I know only to add PR as usual but you asked to add PR to tree or branch which is new for me.
Can you help me to add it plz 
https://github.com/alhitary/upload/tree/patch-1
sorry for this inconvenience

**Comment by lavigor** [comment]
Hi, @alhitary.
I'll have access to my PC at the weekend.
You could switch the base branch by clicking on the corresponding select field just when the PR is created. I'll add the translation a bit later.
Thank you!

File: language/ar/upload.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+	'EXTENSION_UPLOADED_ENABLE'			=> 'تفعيل هذه الإضافة',
+	'ACP_UPLOAD_EXT_UNPACK'				=> 'فك الضغط',
+	'ACP_UPLOAD_EXT_CONT'				=> 'محتوى المجلد : %s',
+
+	'EXTENSION_DELETE'					=> 'حذف الإضافة',
+	'EXTENSION_DELETE_CONFIRM'			=> 'هل تريد بالفعل حذف الإضافة “%s” ؟',
+	'EXTENSIONS_DELETE_CONFIRM'			=> array(
+		2	=> 'هل تريد بالفعل حذف عدد <strong>%1$s</strong> إضافات ؟',
```
Comment (lavigor) [inline language/ar/upload.php]: @alhitary, I think you should read this guideline: https://wiki.phpbb.com/Plural_Rules.
Arabic language has plural rule number 12. Please add other lines as described [here](https://wiki.phpbb.com/Plural_Rules#How_to_use_the_rules).

This comment also applies to other occurrences of this rule. I'll add a note about that to language files later.

**Comment by lavigor** [comment]
@alhitary, now you can edit your translation [here](https://github.com/BoardTools/upload/tree/ar/ar).

**Comment by alhitary** [comment]
I try and try and try and I don't know if do it right or failed @_@
Plz help me to do it right as I'm confused in this new process for me.

I edited and send PRs for each file. And added issue.

https://github.com/BoardTools/upload/issues/30

Thanks

**Comment by alhitary** [comment]
Dear @lavigor
I edited the plural for Arabic language in the requested file.
Plz check now if it is OK
Thanks

**Pull request closed without merging**